kebechet/managers/version/version.py

```diff
@@ -299,14 +301,15 @@ def _compute_changelog(
             ).splitlines()
 
         if version_file:
-            # TODO: We should prepend changes instead of appending them.
             _LOGGER.info("Adding changelog to the CHANGELOG.md file")
-            with open("CHANGELOG.md", "a") as changelog_file:
+            with open("CHANGELOG.md", "r+") as changelog_file:
+                content = changelog_file.read()
+                changelog_file.seek(0, 0)
```

Comment:
Will this work with the title that is present in CHANGELOG file?

Reply:
Oh I see I don't think it'll work with a title, it'll prepend to the front of the file